src/main/java/com/kruize/recommendations/runtimes/java/openj9/OpenJ9JavaRecommendations.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.kruize.recommendations.runtimes.java.openj9;
+
+import com.kruize.recommendations.runtimes.java.JavaRecommendations;
+
+public class OpenJ9JavaRecommendations implements JavaRecommendations
+{
+    /* TODO Also add nursery recommendations and GC */
+    private double heapRecommendation;
+    private double nonHeapRecommendation;
```

Comment:
I would suggest adding the GC Policy here as well. We can set it to `gencon` just to be explicitly clear about which one we support for now.

Reply:
Added GC Policy recommendation and getters and setters for it.